src/chai/ChaiMacros.hpp

```diff
@@ -70,23 +70,33 @@
 
 #define CHAI_UNUSED_ARG(X)
 
+#if !defined(CHAI_DISABLE_ALL_LOGGING)
 #if !defined(CHAI_DISABLE_RM)
```

Comment:
Should this still be disabled if the resource manager is disabled?  Umpire is required either way, right?